static/app/components/group/groupSummary.tsx

```diff
@@ -72,13 +75,24 @@ function GroupSummaryFeatureBadge() {
   );
 }
 
-export function GroupSummaryHeader({groupId}: GroupSummaryProps) {
+export function GroupSummaryHeader({
+  groupId,
+  groupCategory,
+  groupTitle,
+}: GroupSummaryProps) {
   const {data, isLoading, isError, hasGenAIConsent} = useGroupSummary(groupId);
   const isStreamlined = useHasStreamlinedUI();
 
-  if (isError || !hasGenAIConsent || (!isLoading && !data?.headline)) {
+  if (
+    isError ||
+    !hasGenAIConsent ||
+    (!isLoading && !data?.headline) ||
+    groupCategory !== IssueCategory.ERROR ||
+    groupTitle === 'User Feedback'
```

Comment:
Interesting, do we not have a category for user feedback and it's an error?
I'l defer to @malwilley to review this one

Reply:
Yeah some User Feedback issues were classified as ERROR, probably because you can get user feedback when an error occurs in addition to the standalone widget.

I think we still want to hide issue summaries for those, right? So I put this extra check.

Reply:
I think we would want to show issue summaries for those since it still has a stacktrace attached?

Reply:
That's a good point, sometimes it does have a stacktrace. But there's also cases where it doesn't and instead links to a trace connected issue that does have the error.

<img width="559" alt="Screenshot 2024-08-26 at 11 54 51 AM" src="https://github.com/user-attachments/assets/def586c7-d9ff-4a6f-9b01-18fdc8ef8487">

Reply:
In this case I believe it should always be `issueCategory: 'feedback'`, which should be handled by the line above

Reply:
There is no feedback category, so it comes back as "error" in my testing. @malwilley 

![Screenshot 2024-08-26 at 12 03 43 PM](https://github.com/user-attachments/assets/21454dff-b045-4d79-9c8d-cbba52b6c394)

Reply:
Hmm if you're seeing that in prod can you link it? I don't think that's expected.  [This for example](https://sentry.sentry.io/issues/5736361003/) has `issueCategory: 'feedback'` (coming from the backend, the frontend enum may need to be updated to include that value).

Either way not that big of a deal, just would be nice to remove logic checking for a specific title

Reply:
Hmm well I don't see it in prod, removing the title check then